external/kafka-0-8/src/main/scala/org/apache/spark/streaming/kafka/KafkaCluster.scala

```diff
@@ -231,7 +231,10 @@ class KafkaCluster(val kafkaParams: Map[String, String]) extends Serializable {
   // this 0 here indicates api version, in this case the original ZK backed api.
   private def defaultConsumerApiVersion: Short = 0
 
-  /** Requires Kafka >= 0.8.1.1.  Defaults to the original ZooKeeper backed api version. */
+  /**
+   * Requires Kafka higher than or equal to 0.8.1.1.
```

Comment:
Nit: might say this as "Requires Kafka 0.8.1.1 or later". Also could you fix api -> API while you're at it?